base/compiler/abstractinterpretation.jl

```diff
@@ -627,6 +630,16 @@ function is_effect_overrided(linfo::MethodInstance, effect::Symbol)
 end
 is_effect_overrided(method::Method, effect::Symbol) = getfield(decode_effects_override(method.purity), effect)
 
+function override_edge_effects(edge_effects::Effects,
+    sv::InferenceState, method::Method, effect::Symbol)
+    if is_effect_overrided(sv, effect)
+        return override_effect(edge_effects, effect, ALWAYS_TRUE)
+    elseif is_effect_overrided(method, :consistent)
+        return override_effect(edge_effects, effect, ALWAYS_TRUE)
```

Comment:
I don't understand this logic

Reply:
A silly typo. I meant `is_effect_overrided(method, effect)` here.